exporter/sumologicexporter/sender.go

```diff
@@ -92,35 +110,39 @@ func (s *sender) send(ctx context.Context, pipeline PipelineType, body io.Reader
 	// Add headers
 	switch s.config.CompressEncoding {
 	case GZIPCompression:
-		req.Header.Set("Content-Encoding", "gzip")
+		req.Header.Set(headerContentEncoding, contentEncodingGzip)
 	case DeflateCompression:
-		req.Header.Set("Content-Encoding", "deflate")
+		req.Header.Set(headerContentEncoding, contentEncodingDeflate)
 	case NoCompression:
```

Comment:
Just to be sure: this is intended to do nothing, right? As Go requires explicit `fallthrough`

Reply:
Yes, thats correct